### PR TITLE
TCVP-2739 updated FormRecognizer model

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

TCVP-2739
This PR contains the generated files obtained by further training the FormRecognizer 3.0 custom-template model, tagged with verison VT2.1.

Although not perfect, this model is much improved over the previous 2.0 model. Checkboxes appear to be read correctly most of the time, though it does still make mistakes or is unable to read the ticket.

The generated model is now maxed out with training data. To train further, we'd need to replace existing data with an alternative dataset. Custom-template training is limited by the size of the training set.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by karahanb — APPROVED** [review]
Great work!
